front/app/containers/Admin/sideBar/index.tsx

```diff
@@ -8,7 +8,7 @@ import { withRouter, WithRouterProps } from 'utils/cl-router/withRouter';
 
 // components
 import { Icon, IconNames } from '@citizenlab/cl2-component-library';
-import FeatureFlag from 'components/FeatureFlag';
+import FeaturedMenuItem from './FeaturedMenuItem';
```

Comment:
I think we can be smarter about this. Instead of importing 2 MenuItem components, we can add the feature flagging functionality to the MenuItem component.

Reply:
Lmk if you want to pair on that next week.

Reply:
The same reasoning as here https://github.com/CitizenLabDotCo/citizenlab/pull/2106/files/a3f2b8dc6705b6bfc9a6adc5bea8f4f1b1e07236#discussion_r889000780

Also, it would make the hook `useSomeFeatureFlags` to be called under condition (possibly, can be worked around).

Reply:
Here too, it's more a config option and it's about rendering logic, so would keep in the same component. It becomes messy to have check multiple imports and you may miss this "FeaturedMenuItem" component and add separate feature flags where you import MenuItem.

Again, something we could discuss, but I wouldn't introduce this here. I'd keep the feature flag check inside MenuItem.

Reply:
(fixed)